src/Cassandra/Metadata.cs

```diff
@@ -422,16 +422,84 @@ internal void ClearAggregate(string keyspaceName, string aggregateName, string[]
             }
         }
 
+        /// <summary>
+        /// Initiates a schema agreement check. This method does not perform retries so
+        /// <see cref="ProtocolOptions.MaxSchemaAgreementWaitSeconds"/> does not apply. 
```

Comment:
We could include a small paragraph to detail what we mean by "schema agreement", something like:
> Schema changes need to be propagated to all nodes in the cluster. Once they have settled on a common version, we say that they are in agreement.